check/check.go

```diff
@@ -180,7 +179,9 @@ func (c *Check) run() State {
 		}
 	}
 
-	if finalOutput == nil {
+	if finalOutput != nil {
+		glog.V(3).Infof("Check.ID: %s Command: %q TestResult: %t Score: %q \n", c.ID, lastCommand, finalOutput.testResult, c.State)
```

Comment:
I see that you want to print score but you pass `c.State` as arg?

Reply:
Yes, that's a typo. Good catch!